### DISCUSSION TITLE
如何完整裝好模組？

### DISCUSSION THREAD

**Discussion by andy40403** [Meloong-Git/PCL#2058] [提问] [answered]
### <hr>我知道這問題或許對某些人來說很簡單啦～<br>但我不會實裝，只會下載啊！<hr>就我目前的理解，我知道要先這樣：<br>![image](https://github.com/Hex-Dragon/PCL2/assets/66901820/a54d3cc7-9062-4ab0-9611-e3755d66722d)<br>就是先裝個原版還有FORGE或FABIC，然後把模組檔案放進去，<br>那我現在會下載也知道有的要裝前置，<br><a href="https://imgbox.com/sT3M0r5s" target="_blank"><img src="https://thumbs2.imgbox.com/de/3b/sT3M0r5s_t.png" alt="image host"/></a><br>那到FORGE就可以看到這樣，已經下載五十個左右了，<br><a href="https://imgbox.com/n3PxGX0R" target="_blank"><img src="https://thumbs2.imgbox.com/2c/04/n3PxGX0R_t.png" alt="image host"/></a><br>啟動這個1.20FORGE在主選單的樣子：<br>![image](https://github.com/Hex-Dragon/PCL2/assets/66901820/514f94e1-f9cb-4565-a21c-f840a0727b67)<br>可以看到有模組按鈕左下角也有顯示已載入，<br>可是我打開1.20原版卻是這樣！<br>![image](https://github.com/Hex-Dragon/PCL2/assets/66901820/10a9211f-a3b9-400d-b7af-3dbaf534ebe7)<br>啊我也有開啟版本隔離：<br>![image](https://github.com/Hex-Dragon/PCL2/assets/66901820/1182b7ac-c85c-433b-ae3a-cdadb1cd5446)<br>實在搞不懂！<br>我高級長這樣：<br>![image](https://github.com/Hex-Dragon/PCL2/assets/66901820/d7c6731c-7795-4890-a022-41beaccf1199)<br>而且版本都是1.20.0，兩個都沒有哪個是1.20.1的哦！

**Comment by Chiloven945** [comment]
意思是為什麼裝了模組原版卻沒有嗎？
是這樣一個意思，模組版本和原版是分開的，可以理解為多開了幾個遊戲
想要用mod直接啟動那個裝了mod的版本就可以了，因為自動下載是直接下載的遊戲本體然後再自動安裝了加載器

**Comment by WForst-Breeze** [comment, marked as answer]
您的所有安裝和操作步驟都是正確無誤的，而且加裝了 Forge 核心的版本也可以順利啟動，這代表您的模組無沖突。
但是請註意，您安裝的原版遊戲是沒有辦法加載您安裝的模組的，因為您開啟了 **「僅隔離可安裝 Mod 的版本」**，所以除了原版之間可以互通，您安裝的其他有模組加載器的版本，諸如 Forge、Fabric、Quilt、LiteLoader 等，都是沒辦法互通的，他們都是相互獨立存在的，互不影響。
如果您想使得您安裝的模組可以在多個版本中共用，不妨試試在您想要互通的 **有模組加載器的版本** 中關閉版本隔離，並將您下載好的 Mod 都從 **.minecraft/versions/遊戲版本名/mods** 遷移到 **.minecraft/mods** 中去！
![image](https://github.com/Hex-Dragon/PCL2/assets/110760354/090007f0-842e-4640-96ea-62a51b7f413d)
![image](https://github.com/Hex-Dragon/PCL2/assets/110760354/6b07b51a-7ddb-40f0-80b4-f96188c33a98)

**Reply by andy40403** [reply]
### 真的非常謝謝您！<br>最後我乾脆就這樣玩了！<br>
![image](https://github.com/Hex-Dragon/PCL2/assets/66901820/b952e159-15b0-4de1-90c8-57b34232d277)

**Reply by WForst-Breeze** [reply]
> 那請問一下，您啟動頁的網址是啥？我想換成那個！我現在是這樣：

是 Light Beacon 製作的新聞主頁：https://news.bugjump.net

**Reply by iPodtouch6557** [reply]
~保证崩溃启动器~

**Reply by andy40403** [reply]
啥呀😂

**Reply by iPodtouch6557** [reply]
他的自定义窗口标题（翻译可能不准确）

**Reply by andy40403** [reply]
###  @iPodtouch6557  我英文不好當下沒看懂😂
![image](https://github.com/Hex-Dragon/PCL2/assets/66901820/ee8532ec-d178-4f84-8213-ca60e72b8c97)

**Reply by iPodtouch6557** [reply]
行了别谈了
